okapi-core/src/main/java/org/folio/okapi/util/TenantInstallOptions.java

```diff
@@ -69,12 +69,23 @@ public boolean getDepCheck() {
     return depCheck;
   }
 
-  public void setInvoke(boolean v) {
+  public void setInvoke(String v) {
     invoke = v;
   }
 
-  public boolean getInvoke() {
-    return invoke;
+  /**
+   * Check if module is to be invoked during install.
+   * @param id module ID.
+   * @return whether to invoke the module.
+   */
+  public boolean checkInvoke(String id) {
+    if (invoke == null || "true".equals(invoke)) {
+      return true;
+    }
+    if ("false".equals(invoke)) {
```

Comment:
Likewise, do we want to try and handle False / FALSE /etc. here?  maybe via something like `"false".equalsIgnoreCase(invoke)`

Reply:
:-)